.cirrus.yml

```diff
@@ -90,25 +115,22 @@ build_task:
 qa_os_win_task:
   <<: *WINDOWS_VM_DEFINITION
   <<: *ONLY_IF_SONARSOURCE_QA
+  <<: *MAVEN_CACHE
   depends_on:
     - build
-  maven_cache:
-    #windows cache is buggy if using ${CIRRUS_WORKING_DIR}
-    folder: ~/.m2/repository
   build_script:
     - source cirrus-env CI
     - mvn.cmd clean verify
 
 ws_scan_task:
   depends_on:
     - build
-  <<: *LINUX_1_CPU_2G
+  <<: *LINUX_3_5_CPU_7G
+  <<: *MAVEN_CACHE
   # run only on master and long-term branches
   only_if: $CIRRUS_USER_COLLABORATOR == 'true' && ($CIRRUS_BRANCH == "master" || $CIRRUS_BRANCH =~ "branch-.*")
```

Comment:
Why not use here `ONLY_IF_SONARSOURCE_QA`? 
It should remove the warning on GitHub:
```
.cirrus.yml#L174
task "promote" depends on task "ws_scan", but their only_if conditions are different
```